book/src/sec-4-wql.md

```diff
@@ -51,25 +51,25 @@ Updates the content by replacing the previous entity map in entity tree key `my_
 
 ### UPDATE CONTENT
 
-Updates the content by numerical addition or string concatenation of the previous entity map in entity tree key `my_entity_name` with the entity id `48c7640e-9287-468a-a07c-2fb00da5eaed`. Non numerical or non string value will just be replaced. If key doesn't  exist it will be created.
+Updates the content by numerical addition or string concatenation of the previous entity map in entity tree key `my_entity_name` with the entity id `48c7640e-9287-468a-a07c-2fb00da5eaed`. Non numerical or non string value will just be replaced. If the key doesn't exist it will be created.
 
 * `UPDATE my_entity_name CONTENT {a: -4, b: 32,} INTO 48c7640e-9287-468a-a07c-2fb00da5eaed` this will add `-4` to entity map key `a` and add `32` to entity map key `b` in the current entity map stored in entity id `48c7640e-9287-468a-a07c-2fb00da5eaed`.
 
 ### MATCH UPDATE
 
-Similar to SET, but it requires a pre-condition to be satisfied.
+Similar to SET, but it requires a precondition to be satisfied.
```

Comment:
This is something that should be satisfied:
Pre-condition is a statement or set of statements that outline a condition that should be true when an action is called. The precondition statement indicates what must be true before the function is called.

if you pass no codition it is ok

Reply:
I just changed because of english translation, I don't think there is "pre-condition".

Reference: https://translate.google.com/?sl=pt&tl=en&text=pr%C3%A9%20condi%C3%A7%C3%A3o%0A%0Acondi%C3%A7%C3%A3o%20pr%C3%A9via&op=translate

Reply:
translate is limited kkkk but OK